src/app/login-form/login-form.component.html

```diff
@@ -0,0 +1,25 @@
+<mat-card-title>Login</mat-card-title> <span id='invalidText' *ngIf="!hideInvalidText" [(ngModel)]="errorMessage" ngDefaultControl><h2>{{errorMessage}}</h2></span>
+  <mat-card-content>
+    <mat-form-field id="loginUsernameField">
+      <input matInput placeholder="Username" [formControl]="usernameFormControl" [(ngModel)]="usernameLoginText" maxLength="25" >
+      <mat-error *ngIf="usernameFormControl.hasError('required')">
+        Username is <strong>required</strong>
+      </mat-error>
+    </mat-form-field>
+
+    <mat-form-field id="loginPasswordField">
+      <input matInput placeholder="Password" type="password" [formControl]="passwordFormControl" [(ngModel)]="passwordLoginText" maxLength="25">
+      <mat-error *ngIf="passwordFormControl.hasError('required')">
+      Password is <strong>required</strong>
+      </mat-error>
+    </mat-form-field>
+
+  </mat-card-content>
+
+  <mat-card-actions>
+      <mat-checkbox [(ngModel)]="rememberMeCheckbox">Remember Me</mat-checkbox>
```

Comment:
I think this remember me should be moved to the mat card content at least for mobile:
![image](https://user-images.githubusercontent.com/11811793/38451577-a93825c2-39ef-11e8-9567-e4d87b890ccf.png)

Reply:
Ah okay, agreed. I didn't really think about it from a mobile stance.

Reply:
There is a pretty good mobile and resolution viewer in Chrome you can check out 

https://developers.google.com/web/tools/chrome-devtools/device-mode/